app/classifier/tasks/dropdown/dropdown-dialog.cjsx

```diff
@@ -182,6 +230,34 @@ DropdownDialog = React.createClass
 
       <br />
 
+      <div>
+        <h2 className="form-label">Presets</h2>
+        <select ref="preset" onChange={@handlePresetChange}>
```

Comment:
We're already setting state for `numberRange`, why not just keep the value of this select as state, since we eventually change it directly?